lib/Makefile.include

```diff
@@ -34,6 +34,20 @@ STANDARD_FLAGS ?= -std=c99
 
 all: $(SRCLIBDIR)/$(LIBNAME).a
 
+GENFILES = \
+	$(TOPDIR)/include/libopencm3/$(TARGET)/nvic.h \
+	$(TOPDIR)/include/libopencmsis/$(TARGET)/irqhandlers.h \
+	$(TOPDIR)/lib/$(TARGET)/vector_nvic.c
```

Comment:
now we can't have any other generated files?  this doesn't seem scalable or easy for anyone else to ever work on.  (And if you _are_ making lists, use += instead of \ line extensions so that adding entries doesn't cause unrelated changes.

Reply:
ack, `+=` is obviosly better here.

But I don't understand your point about not being able to "habe other generated files"? If new generated (header) files come around we just have to add them to the list.

Reply:
I just meant that now the list of files is here, _and_ inside the generator, and not a lot of visibility that you need to edit both.